src/components/GameAbout/GameAbout.module.scss

```diff
@@ -1,32 +1,119 @@
+.container {
+  display: flex;
+  width: 75rem;
+  padding: var(--space-2xl) 7.5rem;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--space-lg);
+
+  @media (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 100%;
+    padding: var(--space-lg) var(--space-lg);
+  }
+}
+
 .iconOpen {
   transform: rotate(180deg);
 }
 
 .clampedText {
   display: -webkit-box;
+  -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
-  -webkit-line-clamp: 6;
   overflow: hidden;
 }
 
+.title {
+  color: var(--color-neutral-200);
+  margin: var(--space-md) 0;
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.titleLarge {
+  color: var(--color-neutral-200);
+  margin: var(--space-xs) 0;
+  font-weight: 600;
+}
+
+.titleSmall {
+  color: var(--color-neutral-200);
+  margin: var(--space-xs) 0;
+  font-weight: 600;
```

Comment:
worried that we're having to set the font weight here instead of using a typography type. is there a discongruity between our FE design system and figma?

Reply:
see my comment above